docs/sources/source/nlp/dialog_state_tracking_trade.rst

```diff
@@ -32,15 +32,15 @@ multiple turns, possibly spanning over several different domains.
 The MultiWOZ Dataset
 --------------------
 
-The Multi-Domain Wizard-of-Oz dataset (`MultiWOZ`_) is a collection of human-to-human conversations spanning over \
+The Multi-Domain Wizard-of-OZ dataset (`MultiWOZ`_) is a collection of human-to-human conversations spanning over \
 7 distinct domains and containing over 10,000 dialogues.
 The original MultiWOZ 2.0 dataset was introduced in :cite:`nlp-dst-budzianowski2018multiwoz`.
-However, in this tutorial we will utilize MultiWOZ 2.1  :cite:`nlp-dst-eric2019multiwoz`, which aimed at fixing \
-several issues with the original dataset (state errors and corrections, utterance corrections, value 
-cannonicalization etc.).
+However, in this tutorial we will utilize MultiWOZ 2.1  :cite:`nlp-dst-eric2019multiwoz`, which is an updated version\
+ of MultiWOZ 2.0. They have fixed several issues with the original dataset including errors in states, utterances, value
+canonicalization etc.). Our model can also get trained on MultiWOZ 2.0.
```

Comment:
Can you please add description how to do it? Is it difficult to switch to MultiWOZ 2.0?

If not, then it would be great to explain this :+1:

Reply:
They should just change the source and target folder. They are the same. Added a note.